subprojects/core-api/src/main/java/org/gradle/api/Task.java

```diff
@@ -774,7 +772,6 @@ public String determineName(Task c) {
      * @since 5.0
      */
     @Internal
-    @Optional
```

Comment:
Why shouldn't this be `@Optional`? We might not fail without the annotation, but the value _is_ _nullable_ (which `@Optional` actually means). We use `null` to express "no timeout."

Reply:
The problem is, that with `@Optional` with this change I get a report that this field is `@Internal` and `@Optional` which is not allowed. Is `@Internal` annotation wrong here?

Reply:
You are right, I missed the `@Internal` part. With that you don't need `@Optional` indeed.